docs/librarymanager.md

```diff
@@ -0,0 +1,491 @@
+
+For each library or file, the user can edit or delete it. Editing an FXML will open it in Scene Builder. Editing a jar file will open a dialog where the user can preview and select or unselect different components (if any) of that jar that will be added to the Custom panel.
+
+On the lower part of the dialog, there are different actions that the user can perform.
+
+<h4>Search repositories option</h4>
+
+The user can type a name of a library: full name or part of its group id, or full name or part of it of the artifact id, following the usual naming convention.
+
+<p align="center"><img src="images/librarymanager/SceneBuilder08.png" alt="Search Library in Repositories" width="400"></p>
+
+For instance, to download and install the latest release of ControlsFX, just by typing its maven group "org.controlsfx", the existing artifacts (in the registered repositories) will show up:
+
+<p align="center"><img src="images/librarymanager/SceneBuilder09.png" alt="ControlsFX results" width="400"></p>
+
+Then selecting one of the presented results will download the latest release of the artifact from one of the repositories, install it to the user's local .m2, find all possible custom controls found in the jar, and present them for a preview:
+
+<p align="center"><img src="images/librarymanager/SceneBuilder10.png" alt="ControlsFX custom controls" width="400"></p>
+
+The user can browse over the list of custom controls, select anyone for a preview (not all have one), and select one or more custom controls to be imported.
+
+After pressing Import Components, the Import dialog will close and the Manager will show a new entry added:
+
+<p align="center"><img src="images/librarymanager/SceneBuilder11.png" alt="ControlsFX successfully imported" width="400"></p>
+
+And while the full jar was imported, only the selected custom controls are added into the Custom section, ready to be used:
+
+<p align="center"><img src="images/librarymanager/SceneBuilder12.png" alt="Using ControlsFX imported controls" width="400"></p>
+
+If other controls are needed from this jar, the user can open the Library Manager again, and edit the related entry, to select again from the Import dialog.
+
+Also note that if an artifact is removed from the list, it will just be removed from Scene Builder, but the artifact won’t be removed from the local .m2 repository.
+
+<h4>Manually add Library from repository option</h4>
+
+The user can type the exact names of groupID and artifactID coordinates of an artifact to get all existing versions of an artifact in all registered repositories:
+
+<p align="center"><img src="images/librarymanager/SceneBuilder13.png" alt="Add library from repository dialog" width="400"></p>
+
+For instance, for "org.controlsfx" group and "controlsfx" artifact, a list is found in the registered repositories (Maven Central releases, Sonatype snapshots, Local .m2, Nexus, …):
```

Comment:
Do we want to replace double-quotes with backquotes?

Reply:
done